app/controllers/api/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::GreetingsController < ApplicationController
+  def random_greeting
+    random_message = Message.order('RANDOM()').first
+    render json: { greeting: random_message&.message }
+  end
```

Comment:
- [ ] In the database schema the table name is `greeting` and here you're trying to fetch greetings from the `message` table. Please make sure to fix this issue.

Reply:
Dear @thecodechaser 
Thanks for the suggestion, I appreciate it, 
I solved it .
Best wishes,